components/raft_log_engine/Cargo.toml

```diff
@@ -34,5 +34,5 @@ serde_derive = "1.0"
 coarsetime = "0.1"
 kvproto = { git = "https://github.com/pingcap/kvproto.git", default-features = false }
 raft = { version = "0.6.0-alpha", default-features = false }
-raft-engine = { git = "https://github.com/tikv/raft-engine"}
+raft-engine = { git = "https://github.com/hicqu/raft-engine", branch = "rewrite-and-sync" }
```

Comment:
Is this dependency expected?

Reply:
It will be restored after that PR is merged.